repository/n.json

```diff
@@ -1054,6 +1054,18 @@
 				}
 			]
 		},
+		{
+			"name": "Notr",
+			"details": "https://github.com/cepthomas/Notr",
+			"labels": ["note", "note taking", "markdown"],
+			"releases": [
+				{
+					"sublime_text": ">=4000",
+					"platforms": ["windows", "linux"],
```

Comment:
Is this not Mac-compatible now too? If you can't test that's fine, most package devs can't test more than one platform. But if the intention or expectation is that it _could_ work _mostly_, that's good enough 😄

Reply:
Yes. Done.